pybossa/view/account.py

```diff
@@ -499,6 +500,9 @@ def create_account(user_data, project_slugs=None, ldap_disabled=True, auto_creat
                      password=user_data['password'])
     msg = generate_invitation_email_for_new_user(user=user_info, project_slugs=project_slugs)
     mail_queue.enqueue(send_mail, msg)
+    if auto_create is True:
+        admin_msg = generate_bsso_account_notification(user=user_info, admins_emails=['tbarrett@bloomberg.net'], access_type="BSSO")
```

Comment:
should this to be all admins? if not, a particular admin account to be configured and used instead of hardcoding.

Reply:
This is just Tim. Should I create a variable in the local_settings files with his email for use here?